pkg/netpol/connlist/connlist.go

```diff
@@ -412,15 +427,22 @@ func (ca *ConnlistAnalyzer) getConnectionsList(pe *eval.PolicyEngine, ia *ingres
 	}
 
 	// compute connections between peers based on pe analysis of network policies
-	peersAllowedConns, err := ca.getConnectionsBetweenPeers(pe, peers)
+	// if exposure-analysis is on, also compute the exposures-map
+	exposuresMap := exposureMap{}
```

Comment:
why initializing from outside `exposuresMap` instead of returning this map from `getConnectionsBetweenPeers`?

Reply:
done [commit](https://github.com/np-guard/netpol-analyzer/pull/296/commits/b419627fdf18e1e4358605879aa4eef657a9a78d)